packages/vector_graphics_compiler/lib/vector_graphics_compiler.dart

```diff
@@ -96,10 +98,14 @@ void _encodeShader(
 Future<Uint8List> encodeSvg(
     {required String xml,
     required String debugName,
-    bool? enableMaskingOptimizer}) async {
+    bool? enableMaskingOptimizer,
+    bool? enableClippingOptimizer}) async {
```

Comment:
If we're defaulting to true above, and we're not explicitly passing null, could we remove nullability by making these consistently default to `true`?

Reply:
fixed.